packages/components/src/Loader/Loader.md

```diff
@@ -0,0 +1,41 @@
+# **Loader**
+
+# Types of loader 
+
+1. General Loader, sizes are
+    - small
+    - default
+    - large
+    
+2. App Loader (pure css loader)
```

Comment:
I'm not sure this is pure css loader.
I see sass / import so means all resources are already download.
I would see a pure CSS file in my html template with the icon inlined in it.
What do you think about this ?